consensus/istanbul/core/commit.go

```diff
@@ -167,6 +169,16 @@ func (c *core) handleCommit(msg *istanbul.Message) error {
 	return c.handleCheckedCommitForCurrentSequence(msg, commit)
 }
 
+// handleCheckedCommitForPreviousSequence adds messages for the previous
+// sequence to the parent commit set, if the subject digest of msg does not
+// match that of the previous block or it was not sent by one of the previous
+// block's validators an error is returned. If this is the last block of the
+// epoch then the epoch seal will also be validated.
```

Comment:
I found this comment a bit hard to parse at first. Changing up the punctuation would help.
```suggestion
// handleCheckedCommitForPreviousSequence adds messages for the previous
// sequence to the parent commit set. 
// If the subject digest of msg does not match that of the previous block or it
// was not sent by one of the previous block's validators, an error is returned.
// If this is the last block of the epoch then the epoch seal will also be validated.
```

Reply:
I've updated this comment, its slightly different now though because the function has changed.